msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -864,7 +864,7 @@ Delegated permissions allow the app to access data on behalf of the signed-in us
     |`MeetingStage.Write.Chat`|Allows the app to show content on the meeting stage in meetings associated with the chat, on behalf of the signed-in user.|
     |`OnlineMeetingParticipant.Read.Chat`|Allows the app to read participant information, including name, role, id, joined, and left times, of meeting associated with the chat, on behalf of the signed-in user.|
     |`OnlineMeetingParticipant.ToggleIncomingAudio.Chat`|Allows the app to toggle incoming audio for participants in meetings associated with the chat, on behalf of the signed-in user.|
-    |`LiveShareSession.ReadWrite.Chat`|Allows the app to create and synchronize Live Share sessions for meetings associated with the chat. Provide access related information about the meeting's roster, such as member's meeting role, on behalf of the signed-in user.|
+    |`LiveShareSession.ReadWrite.Chat`|Allows the app to create and synchronize Live Share sessions for the chat. Provide access related information about the meeting's roster, such as member's meeting role, on behalf of the signed-in user.|
```

Comment:
For meetings associated with the chat, also provide access related ....

Reply:
Please apply this consistent across all the strings; otherwise, after the update, in its current form, the second sentence is a dangler with no relation to the first sentence.

Reply:
Updated the doc as suggested